src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -105,20 +105,8 @@ export class ReadingsGraphComponent implements OnDestroy {
 
   getTimeBasedAssetReadingsAndSummary(time) {
     this.optedTime = time;
-    if (this.optedTime === 0) {
-      if (this.selectedTab === 4) {
-        this.showAssetReadingsSummary(this.assetCode, this.DEFAULT_LIMIT, this.optedTime);
-      } else {
-        this.plotReadingsGraph(this.assetCode, this.DEFAULT_LIMIT, this.optedTime);
-      }
-    } else {
-      this.limit = 0;
-      if (this.selectedTab === 4) {
-        this.showAssetReadingsSummary(this.assetCode, this.limit, time);
-      } else {
-        this.plotReadingsGraph(this.assetCode, this.limit, this.optedTime);
-      }
-    }
+    this.showAssetReadingsSummary(this.assetCode, this.limit, this.optedTime);
+    this.plotReadingsGraph(this.assetCode, this.limit, this.optedTime);
```

Comment:
Because there are no limit offset things on this modal, now; Hence?

Reply:
Removed